config/routes.rb

```diff
@@ -244,6 +244,9 @@
     get 'terms' => 'terms#index'
   end
 
+  # Relay
+  get ".well-known/social-relay" => "social_relay#well_known"
```

Comment:
Yeah I read that too :) I don't mind applying, but well maybe you are right, it's probably a waste of time. I'll just use `x-` then :)